src/sentry_plugins/splunk/plugin.py

```diff
@@ -241,19 +233,23 @@ def forward_event(self, event, payload):
                 },
             )
 
-            if isinstance(
-                exc,
-                (
-                    ApiHostError,
-                    ApiTimeoutError,
-                ),
+            if isinstance(exc, ApiError) and (
```

Comment:
I find this harder to read than how it was before, I think mainly because of having comments in the middle of a conditional, but it works so I'll approve.